config/config.go

```diff
@@ -0,0 +1,37 @@
+package config
+
+import (
+	"os"
+
+	"gopkg.in/yaml.v3"
```

Comment:
This package is deprecated and the project is archived. This is the continuation and should be a drop in replacement -> https://github.com/yaml/go-yaml

I know that postgres_exporter uses this package and this is copy of that, but I hate to bring in a new dependency that is already deprecated.